# sample node_edge map
EDGE:
    - node_id: [0, 1]
    - node_id: [1, 2]
    - node_id: [2, 3]
    - node_id: [3, 4]
    - node_id: [4, 5]
    - node_id: [5, 6]
    - node_id: [6, 7]
    - node_id: [7, 0]
    - node_id: [0, 8]
    - node_id: [2, 8]
    - node_id: [4, 8]
    - node_id: [6, 8]
MAP_FRAME: '/map'
NODE:
    - id: 0 # 研究室前
     #  type: "intersection"
      type: ""
      point:
           x: 3.029
           y: 1.709
           theta:
      label: ""
      image:
    - id: 1
      type: "respawn"
      point:
           x: 15.654
           y: 12.37
      label: "traffic_detector_launch"
    - id: 2
      type: "intersection"
      point:
           x: 11.377
           y: 17.877
      label: ""
    - id: 3
      type: ""
      point:
           x: 7.052
           y: 23.453
      label: "traffic_detector_launch"
    - id: 4
      type: ""
      point:
           x: -5.849
           y: 12.904
      label: "park"
    - id: 5
      type: "respawn"
      point:
           x: -18.706
           y: 2.285
      label: "traffic_detector_launch"
    - id: 6 # 広い出口
      type: "intersection"
      point:
           x: -14.36
           y: -3.174
      label: "park"
    - id: 7
      type: "intersection"
     #  type: "respawn"
      point:
           x: -9.603
           y: -8.353
      label: "park"
    - id: 8
      type: "intersection"
      point:
           x: -1.048
           y: 7.661
      label: ""
